### PR TITLE
INTG-2699 use the new backend

### PR DESCRIPTION
**Pull request by MickStanciu**
## Checklist

- [ ] Is this PR a reasonable size?

---

**Code Review Guidelines** for Reviewers

- Try to review in a timely manner. Opinions/nitpicks should not be blockers. Pair on a call for non-trivial feedback.
- Overall design and approach should follow established patterns. Don't try to make the PR perfect.
- Try to identify edge cases, race conditions, over-engineering, lack of test coverage and complexity.
- If you don't feel qualified to review the code, pass it on to someone who is.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/SafetyCulture/safetyculture-exporter-ui
 go 1.18
 
 require (
-	github.com/SafetyCulture/safetyculture-exporter v0.0.0-20230306044435-b83b54bbdd0d
+	github.com/SafetyCulture/safetyculture-exporter v0.0.0-20230327025354-362d100ba0e5
```
Comment (sc-gv) [inline go.mod:6]: probably should use the actual version (v.4.6.0) instead of timestamp and commit hash. thoughts?

**Review by sc-gv — APPROVED** [review]
